packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/button.overrides

```diff
@@ -8,7 +8,7 @@
 .ui.button,
 .ui.buttons .button {
     min-height: @scaleHotel;
-    display: flex;
+    display: inline-flex;
```

Comment:
What do you think @fraincs ? With `display: flex` the button itself was not taking 100% of it's parent, but it's container was.

Reply:
all good that's the use of inline-flex